src/fps/utils/Constants.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity 0.8.15;
+
+// Mainnet Chain Ids
+uint256 constant BASE_CHAIN_ID = 8453;
+uint256 constant OPTIMISM_CHAIN_ID = 10;
```

Comment:
nit, we should use `OP_MAINNET` and `OP_SEPOLIA` instead of `OPTIMISM` in these constants